layout/app/catalog/component.js

```diff
@@ -68,18 +116,22 @@ class CatalogLayout extends PureComponent {
                 </div>
               </div>
               <div className="column small-12">
-                <Spinner
-                  className="-light -relative"
-                  isLoading={loading}
-                />
+                <Spinner className="-light -relative" isLoading={loading} />
```

Comment:
If a component has a property, usually you keep it in the same line. More than one, you put them in different lines. Makes code easier to read.